docs/release-notes.asciidoc

```diff
@@ -6,6 +6,9 @@
 
 This section summarizes the changes in each release.
 
+=======
+* <<release-notes-1.5.0>>
+* <<release-notes-1.4.1>>
```

Comment:
Not sure if it is expected that the backport contains release notes for 1.4.1 ?

Reply:
:scream:

Reply:
This is expected, in the same way `1.4` contains `1.3.2`: https://www.elastic.co/guide/en/cloud-on-k8s/1.4/eck-release-notes.html.

Reply:
Should the `1.4.1` related files (`highlights-1.4.1.asciidoc` and `release-notes/1.4.1.asciidoc`) be included in the PR ?
If I try to build the documentation from your branch `backport/1.5.0/pr-4313 ` I get this error:

```
INFO:build_docs:asciidoctor: ERROR: release-notes/highlights.asciidoc: line 27: include file not found: /doc/cloud-on-k8s/docs/release-notes/highlights-1.4.1.asciidoc
INFO:build_docs:asciidoctor: ERROR: release-notes.asciidoc: line 28: include file not found: /doc/cloud-on-k8s/docs/release-notes/1.4.1.asciidoc
```

🤔

Reply:
Absolutely! I completely forgot about it. 

I backported #4302 and #4308.